### DISCUSSION TITLE
[Projects Beta] Navigation within organization or repository

### DISCUSSION THREAD

**Discussion by mkungla** [community/community#10448] [Projects and Issues] [answered]
This missing org and project header navigation in Projects BETA makes it counterproductive to use it as productivity tool when contributors spend most of the time searching their way back to respective repository or organization pages. Simple org/project header like in old version would solve it. However I'm not sure is this feedback repository actively monitored by GitHub. There is countless people who raised this issue months ago, but none of these don't have any feedback from GitHub staff.

There are also very few items on [GitHub public roadmap](https://github.com/orgs/github/projects/4247/views/1?filterQuery=project) addressing project beta issues which none of address that topic. 

**Community has raised that issue already long time ago so what is exactly holding back to give some feedback to these posts?**

List of feedback's I found, most likely there are more:

- https://github.com/github/feedback/discussions/6226
- https://github.com/github/feedback/discussions/6463
- https://github.com/github/feedback/discussions/6560
- https://github.com/github/feedback/discussions/6593
- https://github.com/github/feedback/discussions/6988
- https://github.com/github/feedback/discussions/7530
- https://github.com/github/feedback/discussions/8782
- https://github.com/github/feedback/discussions/9029
- https://github.com/github/feedback/discussions/8889
- https://github.com/github/feedback/discussions/9465
- https://github.com/github/feedback/discussions/9551
- https://github.com/github/feedback/discussions/9597
- https://github.com/github/feedback/discussions/10037
- https://github.com/github/feedback/discussions/10088
- https://github.com/github/feedback/discussions/10443

**found after posting this by community members**

- https://github.com/github/feedback/discussions/8339
- https://github.com/github/feedback/discussions/15094

**Comment by Fiech** [comment]
Great job at compiling these issues! At least a "Back to Projects" navlink should be doable without too much work I assume...

**Comment by dogweather** [comment]
Amazing work, thank you.

**Comment by th0rgall** [comment]
Here are two more related issues:
- https://github.com/github/feedback/discussions/8339
- https://github.com/github/feedback/discussions/15094

**Comment by dogweather** [comment]
Is there a way to opt of Projects Beta? I'd like to disable it organization-wide.

**Reply by matthewferry** [reply]
You can't turn off _just_ the beta, but you can opt out of projects altogether for an org in org settings: `github.com/organizations/{org}/settings/projects`. just uncheck the box to disable projects. This'll disable beta projects _and_ the old project boards.

**Reply by dogweather** [reply]
Ah hah, thanks! This will be useful for a few repos.

**Comment by mkungla** [comment]
Internet has made his voice loud and clear, when will Github start lissening it's users or provide any feedback?

![reactions](https://user-images.githubusercontent.com/15878458/167120285-2f569167-9c2e-4794-a619-efb71c50aca5.jpg)

**Comment by tom-ricci** [comment]
I literally can't use projects without this. Why make a beta project and link it to a repo when you can't access the repo from the project?

**Comment by braymen** [comment]
I was about to create a issue for this, but luckily found this one. This is something that's been a bit of an annoyance and I'd love to just have a simple button or breadcrumb that brings us back to our organization page. It would make navigating a lot easier than going through our profile and all the way back.

**Comment by lostpebble** [comment]
This is a huge annoyance. When I share a project URL link with team members externally (like in Discord etc.) they get to this page, but then there is literally *zero* context as to where they are in Github, and our organization and repos, as a whole. Absolutely terrible UX, and after all these issues and so much feedback about it (I was just about to open a new issue but found _so many_ of them already open) why isn't Github doing something about it?

The project UI should feature clearly: a link to the Organization of the project, and then links to all referenced repos inside the project as well.

**Comment by matthewferry** [comment]
This isn't ideal, but one workaround for this that I've been doing until this is solved more broadly is to keep links to your org and relevant repos in the Project README. I usually include:

- Team repo (where we put the issues for the project)
- Code repos (the repos we're committing to)
- link to the org for quick access

**Comment by 2percentsilk** [comment, marked as answer]
Hey folks!

While we haven't yet fixed the navigation back to repos, we have added breadcrumbs in the header that will let you navigate to the list of projects or the organization view. Check out our [changelog](https://github.blog/changelog/2022-06-02-the-new-github-issues-june-2nd-update/#%F0%9F%A7%AD-improved-navigation-via-project-header).

We know we've still got work to do but hopefully this is a step in the right direction to help you navigate more easily around projects.

**Reply by etienne-napoleone** [reply]
Thanks, that help! Although it looks really different and out of place with the rest of the website. Is it going to be a more generalized change in the future?

**Reply by tom-ricci** [reply]
I'm not sure if this would fit with the new projects design, but can projects be rendered inside the context of where they were viewed like old projects were? For example, if you click on a project from the projects tab in a repo, it would display the project like [this](https://user-images.githubusercontent.com/79022795/172071343-af1624d9-8b39-4304-8e67-eac78cc6b2cf.png), or if you clicked on a project from an organization or a user it would display the organization/user's other tabs. In the sidebar, there could be a list of the org/user who owns the project, the repos, teams, etc. linked to the project, and maybe a button to view the project as a standalone page like it does now.

**Reply by etienne-napoleone** [reply]
I feel like it's part of a future redesign and both systems clash a bit for now
